toml/src/main/kotlin/org/rust/toml/intentions/TomlElementBaseIntentionAction.kt

```diff
@@ -1,21 +0,0 @@
-/*
- * Use of this source code is governed by the MIT license that can be
- * found in the LICENSE file.
- */
-
-package org.rust.toml.intentions
-
-import com.intellij.openapi.editor.Editor
-import com.intellij.openapi.project.Project
-import com.intellij.psi.PsiElement
-import org.rust.ide.intentions.RsElementBaseIntentionAction
-import org.rust.toml.tomlPluginIsAbiCompatible
-
-abstract class TomlElementBaseIntentionAction<Ctx> : RsElementBaseIntentionAction<Ctx>() {
-    final override fun findApplicableContext(project: Project, editor: Editor, element: PsiElement): Ctx? {
-        if (!tomlPluginIsAbiCompatible()) return null
-        return findApplicableContextInternal(project, editor, element)
-    }
```

Comment:
We need this code
It calls `tomlPluginIsAbiCompatible`  that verifies we don't invoke anything it there it runtime incompatibility between toml plugin and rust-toml integration

Reply:
Thank you. I added a `RsTomlElementIntentionAction` which inherits `TomlElementIntentionAction` now